.fixtures.yml

```diff
@@ -13,6 +13,7 @@ fixtures:
     foreman:       "git://github.com/theforeman/puppet-foreman"
```

Comment:
I was able to remove dhcp/dns; however, removing any additional ones at this time does result in test failures.